src/screens/Register.jsx

```diff
@@ -21,21 +32,53 @@ const Register = () => {
   };
 
   const assignUserRole = async (email, values) => {
-    const account = await assignRole({ email, ...values });
-    console.log(account);
+    if (role === "candidate") {
+      const account = await assignCandidate({ email, ...values });
+      console.log(account);
+    } else {
+      const account = await assignCompany({ email, ...values });
+      console.log(account);
+    }
   };
 
-  if (isLoading) return <Loader />;
+  if (isLoading && role) return <Loader />;
 
   return (
     <>
       <PageTitle>
         <h3>Register</h3>
       </PageTitle>
-      <CandidateForm
-        onSubmit={onFormSubmit}
-        fields={CREATE_CANDIDATE_FIELDS}
-      ></CandidateForm>
+      {!role ? (
+        <div style={styleScreen}>
```

Comment:
I wouldn't use CSSinJS here because we keep all our css in `css modules`. Doing CSS in 2 different ways would just be confusing.